api/v1beta2/ibmpowervscluster_types.go

```diff
@@ -32,11 +32,9 @@ const (
 
 // IBMPowerVSClusterSpec defines the desired state of IBMPowerVSCluster.
 type IBMPowerVSClusterSpec struct {
-	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
-	// Important: Run "make" to regenerate code after modifying this file
-
 	// ServiceInstanceID is the id of the power cloud instance where the vsi instance will get deployed.
 	// +kubebuilder:validation:MinLength=1
+	// Deprecated: use ServiceInstance.ID instead
```

Comment:
why just ID? there are other fields like name, regex

Reply:
My intention was to refer ServiceInstanceID to ServiceInstance.ID, But yes we can use anything out of Name, ID, Regex, Update it accrodingly